Makefile

```diff
@@ -89,7 +89,8 @@ generate-swagger:
 	@hash swagger > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
 		$(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger; \
 	fi
-	swagger generate spec -o ./public/swagger.v1.json
+	swagger generate spec -o ./templates/swagger/v1_json.tmpl
+	$(SED_INPLACE) 's;"basePath":\s*"/api/v1";"basePath": "{{AppSubUrl}}/api/v1";g' ./templates/swagger/v1_json.tmpl
```

Comment:
Note for an other PR: We should update the version of the api declare in the json.